Makefile

```diff
@@ -30,16 +30,16 @@ default: dev
 bin: prep
 	@CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS) ui' sh -c "'$(CURDIR)/scripts/build.sh'"
 
+testonly: 
```

Comment:
the changes in the Makefile are incorrect. The correct changes were backported.

Reply:
Good catch! Rebased and popped that commit out.